client/src/components/Toasts/Toast.jsx

```diff
@@ -21,14 +21,24 @@ const NotificationToast = () => {
   );
 
   useEffect(() => {
-    if (toasts.length && !notification) {
-      // Set a new toast when we don't have an active one
-      setNotification({ ...toasts[0] });
-      setTransition(() => (props) => <Slide {...props} direction="right" />);
-      setOpen(true);
-    } else if (toasts.length && notification && open) {
-      // Close an active toast when a new one is added
-      setOpen(false);
+    if (toasts.length) {
+      // avoid showing multiple toasts with same status
```

Comment:
is this needed as there is that `should_notify` in JobsWebhookView

Reply:
Yes I believe you are right that the duplicate issue will already be handled by `should_notify` in JobsWebhookView. I added this part before we simplified the status handling on the server side so this part can essentially be reverted to how it was before.